include/LightGBM/c_api.h

```diff
@@ -839,7 +839,7 @@ LIGHTGBM_C_EXPORT int LGBM_BoosterPredictForCSR(BoosterHandle handle,
  * \param num_iteration Number of iterations for prediction, <= 0 means no limit
  * \param parameter Other parameters for prediction, e.g. early stopping for prediction
  * \param matrix_type Type of matrix input and output, can be ``C_API_MATRIX_TYPE_CSR`` or ``C_API_MATRIX_TYPE_CSC``
- * \param[out] out_len Length of output indices and data
+ * \param[out] out_len Length of output data and indptr (pointer to an array of two elements)
```

Comment:
Thanks for doing this! Maybe replacing `two elements` with `two lengths` will make it more clear, given that `Length` was previously mentioned in this line.

Reply:
I think that'd make it more confusing but I'll try to reword it.